P0009/P0009.md

```diff
@@ -1578,25 +1579,30 @@ template <size_t Rank>
 
    * [1.2]{.pnum} `m` denotes a (possibly const) value of type `M`.
 
-   * [1.3]{.pnum} `E` denotes a specialization of `extents`.
+   * [1.3]{.pnum} `i` and `j` are packs of (possibly const) integers which are multidimensional indices into `m.extents()` ([mdspan.terms]).
```

Comment:
Is the idea that `i` and `j` are in the multidimensional index space represented by `m.extents()`?  If so, then would you consider changing "in" to "into"?
```suggestion
   * [1.3]{.pnum} `i` and `j` are packs of (possibly const) integers which are multidimensional indices in `m.extents()` ([mdspan.terms]).
```

Reply:
ok